Office 365/o365/_meta/smart-descriptions.json

```diff
@@ -54,5 +54,19 @@
                 "field": "event.reason"
             }
         ]
+    },
+    {
+        "value": "User {user.name} performed an action that change status of an investigation. Only automated investigation are logged",
+        "conditions": [
+            {
+                "field": "office365.investigation.type"
+            },
+            {
+                "field": "office365.investigation.name"
+            },
+            {
+                "field": "office365.investigation.status"
+            }
+        ]
```

Comment:
Placeholders in the value field must be present in the conditions, to avoid having description with empty items.
I suggest using [these statuses](https://learn.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-schema#automated-investigation-and-response-events-in-office-365) to offer smartdescriptions.
```suggestion
     },
    {
        "value": "New Investigation {office365.investigation.id}: {office365.investigation.name}",
        "conditions": [
            {
                "field": "office365.investigation.id"
            },
            {
                "field": "office365.investigation.name"
            },
            {
                "field": "office365.investigation.status"
                "value": "Investigation Started"
            }
        ]
    },
    {
        "value": "Investigation {office365.investigation.id} terminated by user {user.name}",
        "conditions": [
            {
                "field": "office365.investigation.id"
            },
            {
                "field": "user.name"
            },
            {
                "field": "office365.investigation.status"
                "value": "Terminated by User"
            }
        ]
    },
    {
        "value": "Investigation {office365.investigation.id}: {office365.investigation.status}",
        "conditions": [
            {
                "field": "office365.investigation.id"
            },
            {
                "field": "office365.investigation.status"
            }
        ]
```